src/components/near-org/NearOrg.Ecosystem.CommunityPage.jsx

```diff
@@ -297,7 +297,7 @@ export function NearOrgEcosystemCommunityPage() {
                   </Text>
                   {item.internal ? (
                     <VmComponent
-                      src="near/widget/DIG.Button"
+                      src={process.env.NEXT_PUBLIC_NETWORK_ID == 'testnet' ? 'discom.testnet/widget/DIG.Button' : 'near/widget/DIG.Button'}
```

Comment:
@milenakrawczyk I think you can simplify this by using this https://github.com/near/near-discovery/blob/main/src/data/bos-components.ts#L41. You can add src of `DIG.Button` for mainnet and testnet. Then you just call
```
const components = useBosComponents();
```
in component you need. And pass src like this 
```suggestion
                      src={components.<yourComponentName>}
```

Reply:
Thanks! This looks much cleaner 👍